src/creatures/monsters/monster.cpp

```diff
@@ -2221,3 +2219,24 @@ void Monster::setMonsterIcon(uint16_t iconcount, uint16_t iconnumber)
 bool Monster::canDropLoot() const {
 	return !mType->info.lootItems.empty();
 }
+
+std::vector<std::pair<int8_t, int8_t>> Monster::getPushItemLocationOptions(const Direction &direction) {
+	if(direction == DIRECTION_WEST || direction == DIRECTION_EAST){
+		return {{0, -1}, {0, 1}};
+	}
+	if(direction == DIRECTION_NORTH || direction == DIRECTION_SOUTH){
+		return {{-1, 0}, {1, 0}};
+	}
+	if(direction == DIRECTION_NORTHWEST){
+		return {{0, -1}, {-1, 0}};
+	}
+	if(direction == DIRECTION_NORTHEAST){
+		return {{0, -1}, {1, 0}};
+	}
+	if(direction == DIRECTION_SOUTHWEST){
+		return {{0, 1}, {-1, 0}};
+	}
+	if(direction == DIRECTION_SOUTHEAST){
+		return {{0, 1}, {1, 0}};
+	}
+}
```

Comment:
It is necessary to return something in the last bracket of the function (and code pattern)
```suggestion
	if (direction == DIRECTION_WEST || direction == DIRECTION_EAST) {
		return {{0, -1}, {0, 1}};
	}
	if (direction == DIRECTION_NORTH || direction == DIRECTION_SOUTH) {
		return {{-1, 0}, {1, 0}};
	}
	if (direction == DIRECTION_NORTHWEST) {
		return {{0, -1}, {-1, 0}};
	}
	if (direction == DIRECTION_NORTHEAST) {
		return {{0, -1}, {1, 0}};
	}
	if (direction == DIRECTION_SOUTHWEST) {
		return {{0, 1}, {-1, 0}};
	}
	if (direction == DIRECTION_SOUTHEAST) {
		return {{0, 1}, {1, 0}};
	}

	return {};
}
```